src/components/Card/Card.module.scss

```diff
@@ -40,28 +41,23 @@
       background: #8f8f8f;
     }
   
-      &Item {
```

Comment:
no idea what have you changed here...tabs? this file has nothing to do with your PR, revert changes and leave this one how it's done on master branch

Reply:
Actually i dont know why row#43 is here.
Because i`ve add one line in very top "box-sizing: border-box;" and delete artificial rows from 61-64
![image](https://github.com/BUI-studies/yidlorama/assets/113294189/08fa833f-5219-46da-828c-59fefd1a7b5d)

i`m understanding, that i shouldnt do it in this PR but we`ve worked in pair with Natalia and check workability of this styles (she asked me this changes)